### DISCUSSION TITLE
Unable to upgrade from 4.2.12 to 4.3.0

### DISCUSSION THREAD

**Discussion by kristophrk** [mastodon/mastodon#32898] [Q&A]
Hey all - 

I'm trying to get my server back up to current release.  I managed to go from 4.2.11 to 4.2.12 just fine today.  But when I try to go to 4.3 all the steps show as successful / no errors.  yet when I restart everything (and do the db migration) and reboot the server - it still shows 4.2.12.

Why is it not going to 4.3?  What step am I missing?  I followed the guide to a t.

**Comment by mjankowski** [comment]
Are you using a fork with any changes, or just regular tagged release?

Are you using Docker deploy or something else?

When you say "reboot the server", do you mean that literally, or do you mean "Restart the web app processes"?

I agree that is whatever you did to bump from 4.2.11 to 4.2.12 worked, and you did roughly that same thing but following guide to go to 4.3 ... that should work!? So I guess ... did you figure this out? If not, add some info about server env along w/ above questions.
